rust/azure_iot_operations_connector/src/base_connector/managed_azure_device_registry.rs

```diff
@@ -730,6 +1068,108 @@ impl DeviceEndpointStatus {
     }
 }
 
+/// Represents the specification of an Asset in the Azure Device Registry service.
+#[derive(Debug)]
+pub struct AssetSpecification {
```

Comment:
Obviously fine to keep this all `pub` for now, but can any of these be invalid?

Reply:
i.e. can these be edited to have values that we wouldn't want it to have?

Reply:
ah, I see - so this struct is under a getter on the AssetClient, so if someone modifies something on it, it isn't tied to the actual Client anymore, it's just their copy that they'd be modifying